### PR TITLE
Add Pibrella

### PR DESCRIPTION
**Pull request by ForToffee**
Using JamHat as template, added Pibrella so can be used remotely via PiZero GPIOExtender

### CODE REVIEW

File: gpiozero/boards.py [hunk 1]
```diff
@@ -2237,3 +2237,98 @@ def off(self):
...
+        :class:`LEDBoard` instance representing the three LEDs  
+
+        .. attribute:: red, yellow, green
+
+            :class:`LED` or :class:`PWMLED` instances representing the red,
+            yellow, and green LEDs .
+
+    .. attribute:: outputs
+    
+        :class:`LEDBoard` instance representing the 4 Out headers on the Pibrella.
```
Comment (lurch) [inline gpiozero/boards.py]: I'm not sure LEDBoard is appropriate here... it's a bit uglier code-wise, but I think a CompositeOutputDevice of DigitalOutputDevices would be better. (and also shows that theses are 'generic' outputs and not 'LED' outputs?)

File: gpiozero/boards.py [hunk 2]
```diff
@@ -2237,3 +2237,98 @@ def off(self):
...
+                              pwm=pwm, _order=('red', 'yellow', 'green'),
+                              pin_factory=pin_factory),
+            inputs=ButtonBoard(a=9, b=7, c=8, d=10,
+                              pull_up = False, _order=('a', 'b', 'c', 'd'),
+                              pin_factory=pin_factory),
+            button=Button(11, pull_up=False, pin_factory=pin_factory),
+            outputs=LEDBoard(e=22, f=23, g=24, h=25,
+                              pwm=pwm, _order=('e', 'f', 'g', 'h'),
+                              pin_factory=pin_factory),
+            buzzer=TonalBuzzer(18, pin_factory=pin_factory),
```
Comment (lurch) [inline gpiozero/boards.py]: Is this definitely a TonalBuzzer (can it play different tones?) or is it just a regular Buzzer?

File: gpiozero/boards.py [hunk 3]
```diff
@@ -2237,3 +2237,98 @@ def off(self):
...
+
+            :class:`LED` or :class:`PWMLED` instances representing the 4 outputs .
+        
+    .. attribute:: button
+
+        The red :class:`Button` objects on the Pibrella.
+
+    .. attribute:: inputs
+    
+        :class:`ButtonBoard` instance representing the 4 In headers on the Pibrella.
```
Comment (lurch) [inline gpiozero/boards.py]: Similarly, these might be better as a CompositeDevice of DigitalInputDevices?

File: gpiozero/boards.py [hunk 4]
```diff
@@ -2237,3 +2237,98 @@ def off(self):
...
+        If :data:`True`, construct :class: PWMLED instances to represent each
+        LED on the board. If :data:`False` (the default), construct regular
+        :class:`LED` instances.
+
+    :type pin_factory: Factory or None
+    :param pin_factory:
+        See :doc:`api_pins` for more information (this is an advanced feature
+        which most users can ignore).
+
+    .. _Pibrella: https://www.pibrella.com/
```
Comment (lurch) [inline gpiozero/boards.py]: I dunno if it's just me, but https://www.pibrella.com/ shows me a totally different website to http://www.pibrella.com/ ?! :confused:

File: gpiozero/boards.py (hunk 2 above)
Comment (ForToffee) [reply gpiozero/boards.py]: Yes, definitely a TonalBizzer (passive not active)

File: gpiozero/boards.py (hunk 4 above)
Comment (ForToffee) [reply gpiozero/boards.py]: Ah, my bad, force of habit to https everything these days.

File: gpiozero/boards.py (hunk 2 above)
Comment (lurch) [reply gpiozero/boards.py]: Ah yes, I've just found https://github.com/pimoroni/pibrella#the-buzzer
:+1:

Comment (ForToffee) [reply gpiozero/boards.py]: I have just noticed that the documentation section of the class refers to Buzzer not TonalBuzzer.  I have fixed this for both the Pibrella and JamHat classes.

File: gpiozero/boards.py (hunk 1 above)
Comment (ForToffee) [reply gpiozero/boards.py]: I have no strong preference,  LEDBoard fits the need (and there are LEDs attached to the outputs) and follows the pattern of the original API.  I'll look at the suggestion you've made, not been that deep into GPIOZero yet :-)

File: gpiozero/boards.py (hunk 3 above)
Comment (ForToffee) [reply gpiozero/boards.py]: As per my comment for the outputs.

File: gpiozero/boards.py (hunk 4 above)
Comment (ForToffee) [reply gpiozero/boards.py]: Fixed

**Comment by bennuttall** [comment]
Closing in favour of #798 - thanks @ForToffee

**Pull request closed without merging**